src/admin/admin.service.ts

```diff
@@ -136,4 +163,44 @@ export class AdminService {
       throw new Error(`Invalid report category: ${value}`);
     }
   }
+
+  async sendTransferRequest(
```

Comment:
This method should not be accessed from outside the class. Please designate it as private, for example: `private async sendTransferRequest`.